Add info on specifying a replica version (#2541)
dfinity/portal: docs/developer-docs/developer-tools/cli-tools/advanced-dfx/specifying-replica-version.mdx

# Specifying the replica version in dfx

## Overview

> The replica version included in the most recent release of `dfx` may not reflect the most up to date replica release. In some workflows, such as testing new features of the replica, having the most recent version of the replica is necessary. 

Reviewer: ```suggestion
The replica version included in the most recent release of `dfx` may not reflect the most up to date replica release. In some workflows, such as testing new features of the replica, having the most recent version of the replica may be desirable. 
```
